packages/orval/src/write-specs.ts

```diff
@@ -132,8 +132,8 @@ export const writeSpecs = async (
         await fs.appendFile(
           indexFile,
           uniq(importsNotDeclared)
-            .map((imp) => `export * from '${imp}';`)
-            .join('\n') + '\n',
+            .map((imp) => `export * from '${imp}';\n`)
+            .join(),
```

Comment:
It should be aligned with the pattern written in else.

https://github.com/orval-labs/orval/blob/419fa4831163a04ee4f129a01d756842ed61139f/packages/orval/src/write-specs.ts#L139-L144

```suggestion
            .map((imp) => `export * from '${imp}';`)
            .join('\n'),
```

Reply:
I was a little worried that if I implemented it with `.join('\n')`, the last line would not have a newline.

```diff
"export * from './zod/pet/pet';\n" +
"export * from './zod/store/store';\n" +
"export * from './zod/user/user';\n" +
+"export * from './types';\n" // my pattern
-"export * from './types';"    // with `.join('\n')`
```

Reply:
I see. You're right.